newIDE/app/src/CompactPropertiesEditor/index.js

```diff
@@ -0,0 +1,791 @@
+            )
+          }
+          disabled={disabled}
+          label={field.label}
+          onClick={() => {
+            field.onClick(instances[0]);
+          }}
+        />
+      );
+    },
+    [instances]
+  );
+
+  const renderResourceField = (field: ResourceField) => {
+    if (!project || !resourceManagementProps) {
+      console.error(
+        'You tried to display a resource field in a PropertiesEditor that does not support display resources. If you need to display resources, pass additional props (project, resourceManagementProps).'
+      );
+      return null;
+    }
+
+    const { setValue } = field;
+    return (
+      <ResourceSelectorWithThumbnail
+        key={field.name}
+        project={project}
+        resourceManagementProps={resourceManagementProps}
+        resourceKind={field.resourceKind}
+        fallbackResourceKind={field.fallbackResourceKind}
+        resourceName={getFieldValue({
+          instances,
+          field,
+          defaultValue: '(Multiple values)', //TODO
+        })}
+        onChange={newValue => {
+          instances.forEach(i => setValue(i, newValue));
+          _onInstancesModified(instances);
+        }}
+        floatingLabelText={getFieldLabel({ instances, field })}
+        helperMarkdownText={getFieldDescription(field)}
+      />
+    );
+  };
+
+  const renderVerticalCenterWithBar = (field: Field) =>
+    field.child && field.child.getValue ? (
+      <VerticallyCenterWithBar key={field.name}>
+        {renderInputField(field.child)}
+      </VerticallyCenterWithBar>
+    ) : (
+      'TODO'
+    );
+
+  const renderContainer =
+    mode === 'row'
+      ? (fields: React.Node) =>
+          preventWrap ? (
+            <LineStackLayout noMargin alignItems="center" expand>
+              {fields}
+            </LineStackLayout>
+          ) : (
+            <ResponsiveLineStackLayout noMargin alignItems="center" expand>
+              {fields}
+            </ResponsiveLineStackLayout>
+          )
+      : (fields: React.Node) => (
+          <ColumnStackLayout noMargin expand>
+            {fields}
+          </ColumnStackLayout>
+        );
+
+  const renderTitle = React.useCallback(
+    (field: Title) => {
+      const { getValue, renderLeftIcon } = field;
+
+      let additionalText = null;
+
+      if (getValue) {
+        let selectedInstancesValue = getFieldValue({
+          instances,
+          field,
+          defaultValue: field.defaultValue || 'Multiple Values',
+        });
+        if (!!selectedInstancesValue) additionalText = selectedInstancesValue;
+      }
+
+      if (!!additionalText) {
+        return (
+          <LineStackLayout
+            alignItems="center"
+            key={`section-title-${field.name}`}
+            expand
+          >
+            {renderLeftIcon()}
+            <Text displayInlineAsSpan noMargin>
+              {field.name}
+            </Text>
+            <Text allowSelection displayInlineAsSpan size="body2" noMargin>
```

Comment:
you allow the selection of this `-` and the text after, but not the field name. Is that on purpose?

Reply:
I removed it, good catch. I think only the object name is interesting